lib/etl.js

```diff
@@ -95,8 +104,20 @@ function subscribe(rippled) {
 
     if (data.type === 'validationReceived') {
       data.reporter_public_key = connections[this.url].public_key;
+
+      // Store master key if validation is signed
+      // by a known valid ephemeral key
+      const master_public_key = manifests.getMasterKey(
+        data.validation_public_key);
+      if (master_public_key) {
```

Comment:
That's an option. My thinking is that we can determine an ephemeral key for a validation by looking at the manifests. And for reports and domain verification, I think we should be identifying validators by their master key.